### PR TITLE
Add missing Spanish template

### PR DESCRIPTION
**Pull request by quique**

### CODE REVIEW

**Comment by baltpeter** [comment]
Thanks for the translation, @quique.

@m-rey Since you also worked on the other Spanish templates, would you mind having a look at this PR?

File: templates/es/objection-default.txt [hunk 1]
```diff
@@ -0,0 +1,15 @@
+A quien corresponda:
+
+Por la presente, me opongo al procesamiento de mis datos personales con fines de marketing directo de acuerdo con el artículo 21 (2) de la GDPR.  Esta oposición incluyte también su uso para la elaboración de perfiles en la medida en que esté relacionada con el marketing directo.
+
+Si hubiera dado mi consentimiento para el procesamiento de mis datos personales con fines de marketing directo (por ejemplo, de acuerdo con el Art. 6(1) o el Art. 9(2) GDPR), por la presente revoco dicho consentiemiento.
```
Comment (m-rey) [inline templates/es/objection-default.txt]: * `GDPR` -> `RGPD`

optional:
* `por ejemplo` -> `p.ej.` (for consistency)
* `Art. 6(1) o el Art. 9(2)` -> `artículo 6(1) o el artículo 9(2) del` (for consistency)

File: templates/es/objection-default.txt [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+Mi solicitud incluye explícitamente [runs>lo siguiente así como ] cualquier otro servicio y compañía de las que usted sea responsable en el sentido del Art. 4(7) GDPR [runs>: {runs_list}].
```
Comment (m-rey) [inline templates/es/objection-default.txt]: * `compañía` -> `empresa` (for consistency)
* `GDPR` -> `GPRD`

optional:
* `Art. 4(7)` -> `artículo 4(7) del`

File: templates/es/objection-default.txt [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+Según lo establecido en el Artículo 12(3) de la GDPR, debe usted confirmarme la objeción sin demora indebida y, en cualquier caso, dentro del mes siguiente a la recepción de la solicitud.
```
Comment (m-rey) [inline templates/es/objection-default.txt]: * `Artículo 12(3) de la GDPR` -> `artículo 12(3) del RGPD`

**Comment by baltpeter** [comment]
Hey @quique, would you mind taking a look at the changes proposed by @m-rey? Thank you!

**Comment by quique** [comment]
Consistency is valuable, so I applied the suggested changes.

**Pull request merged**